src/GeneratedMapper.Tests/CollectionExtensionMethodGeneratorTests.cs

```diff
@@ -26,7 +26,6 @@ public class A {
 
 namespace B {
     public class B { public string[] Prop { get; set; } }
-}
```

Comment:
I saw these issues when writing tests, but didn't bother to fix them as they were working already.  More of a reason to test with normal unit tests as it weed's out invalid syntax being refactored.